### ISSUE TITLE
is there a setting to set the mode to mop or vacuum and mop or just vacuum mode?

### ISSUE THREAD

**Issue by rivendellstuff** [DeebotUniverse/docs#255]
cant seem to find a sensor for it. I have it running a few times a day in the same room where cats litter box is but I dont want it mopping 4 times a day. Would be nice to be able to dynamically control that.

ty

**Comment by edenhaus** [comment]
Yes there is, but currently only activated for T20. If you have another model, please create a capabilities files for you model

**Comment by rivendellstuff** [comment]
its this one from Costco which I believe is the T20 but just labeled differently in Canada.
**"ECOVACS DEEBOT TEO OMNI"**

https://www.costco.ca/ecovacs-deebot-teo-omni-vacuum-%2526-mop-robot-vacuum.product.4000190187.html

**Comment by edenhaus** [comment]
I cannot verify it from the link above.
Please download the diagnostic information of your device in HA and verify that the device class exists as file [here](https://github.com/DeebotUniverse/client.py/tree/dev/deebot_client/hardware/deebot).
If your device class is missing in the folder, than you need to add the capabilities of your bot

**Comment by rivendellstuff** [comment]
thanks.

I'm quite certain its the same capabilities as the T20. but the model class is: 85nbtp

Which doesnt appear to be in that folder.  I dont know how to generate the capabilities file. 

[deebot-teo-omni.txt](https://github.com/DeebotUniverse/docs/files/13313271/deebot-teo-omni.txt)
